README.md

```diff
@@ -368,294 +396,260 @@ A device renderer instance can be configured using the `options` argument (objec
 
-### `mouse`
+#### <img style="vertical-align: middle" src="./doc/assets/ic_camera.svg"> `camera`
 
 -   **Type:** `Boolean`
 -   **Default:** `true`
+-   **Toolbar name:** `Camera`
 -   **Compatibility:** `PaaS`, `SaaS`
 -   **Details:**
-    Enables or disables the mouse events. If you want to disable all VM interaction, please also disable `touch` and `keyboard`.
+    Enables or disables the camera widget. This widget can be used to forward local webcam video to the Android virtual device. By default, if the `microphone` property is also true, then the default audio input will be used as well.
 
-### `keyboard`
+#### <img style="vertical-align: middle" src="./doc/assets/ic_clipboard.svg"> `clipboard`
 
 -   **Type:** `Boolean`
 -   **Default:** `true`
+-   **Toolbar name:** `Clipboard`
 -   **Compatibility:** `PaaS`, `SaaS`
 -   **Details:**
-    Enables or disables the keyboard widget. This widget can be used to transmit keyboard key strokes to the Android virtual device.
+    Enables or disables the clipboard widget. This widget can be used to forward local clipboard to the Android virtual device.
 
-### `keyboardMapping`
+#### <img style="vertical-align: middle" src="./doc/assets/ic_screencast.svg"> `capture`
 
 -   **Type:** `Boolean`
 -   **Default:** `true`
+-   **Toolbar name:** `Screencast`
 -   **Compatibility:** `PaaS`, `SaaS`
 -   **Details:**
-    Enables or disables the keyboardMapping. This widget can be used to map key with command (i.e. tap, swipe-left, tilt, ...).
-
-### `volume`
+    Enables or disables the capture widget. This widget can be used to capture the screen of the Android virtual device (screenshot or screencast).
 
-<img align="right" src="./doc/assets/ic_sound_active_black.svg" alt="..."></img>
+#### <img style="vertical-align: middle" src="./doc/assets/ic_diskIO.svg"> `diskIO`
 
 -   **Type:** `Boolean`
 -   **Default:** `true`
+-   **Toolbar name:** `IOThrottling`
 -   **Compatibility:** `PaaS`, `SaaS`
 -   **Details:**
-    Enables or disables the volume widget. This widget can be used to increase or decrease the volume of the Android virtual device.
-
-### `rotation`
+    Enables or disables the diskIO widget. This widget can be used to modify Disk IO (throttling) of the Android virtual device.
 
-<img align="right" src="./doc/assets/ic_rotation.svg" alt="..."></img>
+#### <img style="vertical-align: middle" src="./doc/assets/ic_installation.svg"> `fileUpload`
 
 -   **Type:** `Boolean`
 -   **Default:** `true`
+-   **Toolbar name:** `FileUpload`
 -   **Compatibility:** `PaaS`, `SaaS`
 -   **Details:**
-    Enables or disables the rotation widget. This widget can be used to rotate the Android virtual device.
-
-### `navbar`
+    Enables or disables the fileUpload widget and drag & drop. This widget can be used to forward local file to the Android virtual device. When drag & dropping APK or ZIP files, it will install them.
```

Comment:
```suggestion
    Enables or disables the fileUpload widget and drag & drop. This widget can be used to forward local file to the Android virtual device. When dragging & dropping APK or ZIP files, it will install them.
```